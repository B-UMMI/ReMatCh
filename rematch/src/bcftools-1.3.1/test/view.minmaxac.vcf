##fileformat=VCFv4.2
##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta
##contig=<ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo sapiens",taxonomy=x>
##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of Samples With Data">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Total Depth">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=HQ,Number=2,Type=Integer,Description="Haplotype Quality">
##FORMAT=<ID=TS,Number=2,Type=String,Description="Test Empty Format String">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA00001	NA00002	NA00003	NA00004
20	1234567	.	A	G,C,T	50	PASS	NS=3;DP=9	GT:GQ:DP:TS	0/1:35:4:A,.	0/2:17:2	1/1:40:3	0/3:17:2
20	1234568	.	A	G,C,T	50	PASS	NS=3;DP=9	GT:GQ:DP:TS	0/1:35:4:A,.	2/2:17:2	3/3:40:3	0/1:17:2
