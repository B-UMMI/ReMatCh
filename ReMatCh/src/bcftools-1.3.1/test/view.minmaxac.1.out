20	1234567	.	A	G,C,T	50	PASS	NS=3;DP=9;AC=3,1,1;AN=8	GT:GQ:DP:TS	0/1:35:4:A,.	0/2:17:2:.	1/1:40:3:.	0/3:17:2:.
